### ISSUE TITLE
Release files for 1.2.1 only contain files for 1.2.0

### ISSUE THREAD

**Issue by FokkeDijkstra** [nmquijada/tormes#33]
I am trying to install TORMES using the github release. Unfortunately the archives for version 1.2.1 only contain bin/version-1.2.0 and not bin/version 1.2.1.
Can you fix this?

Kind regards,

Fokke

**Comment by nmquijada** [comment]
Hi Fokke,

How you are you doing it? For me:
```
git clone https://github.com/nmquijada/tormes.git
ls tormes/bin
# version-1.0  version-1.1  version-1.2.1
sh tormes/bin/version-1.2.1/tormes -v
# TORMES version 1.2.1
```

However, I would recommend to install TORMES as a conda environment, as explained here: https://github.com/nmquijada/tormes#installation
By doing so, every third-party software and their required dependencies and variables will be succesfully set up.

Best,
Narciso

**Comment by FokkeDijkstra** [comment]
Dear Narciso,

I was talking about the the release 1.2.1 tar.gz and zip files. I am actually trying to install all dependencies on our compute cluster to prevent users from having massive private conda installations.
And since I want to use official releases the best way is to fetch the tar.gz file from github.  

Kind regards,

Fokke

**Comment by nmquijada** [comment]
Dear Fokke,

Thank you very much for reporting it! I was not aware that the files from the tag did not match the current version of the pipeline.
I do not have access to my "git" tools right now so I cannot entirely delete a tag by using the GitHub web interface. As an "intermediate" solution, I created a new tag properly matching the version 1.2.1 n the tar.gz and zip files: https://github.com/nmquijada/tormes/releases/tag/v1.2.1-current
I will try to "properly" fix it soon (or at least with the next relase).

I see your point with the environments' managing and your approach is absolutely valid. My only concern about building tormes that way is that the versions of the other dependencies do not differ very much from those tested in the release (included in the tormes-1.2.1.yml). Some software change their output with the novel releases and can compromise tormes output (for instance, the output tables of the new version of [sistr](https://anaconda.org/bioconda/sistr_cmd) had different fields compared to the former version, and tormes-report considers the structure of this last version when taking such fields for the report).
Every time I release a new version of tormes, I test it with the latest versions of all dependencies (and include them in the yaml file), so as far as the other software you have in your system are up-to-date, you will be fine.

Probably, such changes might happen rarely and the chances of the report giving you an "unexpected" output are rare. I just wanted to "warn" you so you have it in mind ;)
I think I can include some "firewalls" in the code to display a warning (in the log file for instance) in case of different dependencies versions.

Hope it works now. Thanks for the call!

Narciso
